packages/sdk-node/src/types.ts

```diff
@@ -33,3 +34,11 @@ export type ClerkMiddlewareOptions = {
   strict?: boolean;
   signInUrl?: string;
 } & MultiDomainAndOrProxy;
+
+export type ClerkClient = ReturnType<typeof Clerk>;
```

Comment:
Are these public apis?

Reply:
@nikosdouvlis No, these are not publicly exported. The top-level index file exports specific types.